eng/common/scripts/modules/ChangeLog-Operations.psm1

```diff
@@ -1,5 +1,5 @@
 # Common Changelog Operations
-
+# This Files has been retired
```

Comment:
If these edits are going to remain please link to an issue with talks about them being retired and removed.